frontend/src/scenes/web-analytics/pageReportsLogic.ts

```diff
@@ -245,6 +252,46 @@ export const pageReportsLogic = kea<pageReportsLogicType>({
                     citiesQuery: getQuery(TileId.GEOGRAPHY, GeographyTab.CITIES),
                     timezonesQuery: getQuery(TileId.GEOGRAPHY, GeographyTab.TIMEZONES),
                     languagesQuery: getQuery(TileId.GEOGRAPHY, GeographyTab.LANGUAGES),
+
+                    // Top events query
+                    topEventsQuery: {
+                        kind: NodeKind.InsightVizNode,
+                        source: {
+                            kind: NodeKind.TrendsQuery,
+                            series: [
+                                {
+                                    kind: NodeKind.EventsNode,
+                                    event: null,
+                                    name: 'All events',
+                                    math: 'total',
+                                },
+                            ],
+                            trendsFilter: {},
+                            breakdownFilter: {
+                                breakdowns: [
+                                    {
+                                        property: 'event',
+                                        type: 'event_metadata',
+                                    },
+                                ],
+                            },
+                            properties: pageUrl
+                                ? [
+                                      createUrlPropertyFilter(pageUrl, stripQueryParams),
+                                      {
+                                          key: 'event',
+                                          value: ['$pageview', '$pageleave'],
```

Comment:
Why not include these? Or why include some other built-in events e.g. `$autocapture`, `$set`, `$identify`, feature flags, etc?

Reply:
I think removing `$pageview` would make sense because it is already on the top chart of "combined metrics," so `$pageleave` was, in a way, the same logic to an extent.

Those other ones felt like they would be worthy to keep because we have many cases like "Are you calling .identify on those pages?".

I am happy not to filter any if you think that would be more useful.

Reply:
Thinking about this a bit more, my take was that since this is lower on the page, it would be contextual to the other data available, which is different than just a one-off insight analysis or general dashboard. Another thing is that fewer series would be a cleaner visualization of what is happening *on* the page.